### DISCUSSION TITLE
Participation at the AC's discussions

### DISCUSSION THREAD

**Discussion by pfink** [openhab/openhab-distro#1492] [Architecture Council Discussions]
In https://github.com/openhab/openhab-docs/issues/1000 we discovered that the ACs discussions on this page are only visible by members of the GitHub openHAB organization. Also, there is no way to see the members of the AC for people outside the GitHub organization. I'd like to open a discussion about this here because in my opinion this is a topic that should not be decided implicitely on how this page was technically set up.

I think it would be crucial that the AC discusses & clearly defines:

* Which group of people should be able to see who are the **members of the AC**?
* Which group of people should be able to follow (**read**) the AC's **discussions**?
* Which group of people is invited to **participate** (do posts & open new topics) at the AC's **discussions**?
* Which group of people should be able to **read** the AC's **decisions**?
* Which group of people should be able to **follow** an **election** of a new AC member (see who did a nomination of a new member as well as discussion / votes of the existing members)?

Personally I could live with staying with the current solution (even if I think it's not ideal as I'd prefer that the AC's members & discussions would be publicly readable). I just think that everybody should be aware of it and know why we do it like this. In my point of view we should build the technical solution for the AC communication / documentation upon the organisational decision and not the other way round.

**Comment by rkoshak** [comment]
I don't have a strong opinion for most of this. The list of members was publicly announced in the forum so I don't think that is sensitive. 

As for the rest, since the purpose of the AC is as a "higher authority" maintainers can appear to if there is disagreement that the maintainers cannot deal with on their own, and to address some cross cutting concerns (e.g. security issue reporting) I don't have too much of a concern if the discussions are kept restricted in some ways. 

I think it would be useful for maintainers to be able to see the discussions and decisions and election stuff. I don't know if it needs to be more public than that, though I don't have a strong opinion on the matter.

**Comment by pfink** [comment]
> since the purpose of the AC is as a "higher authority" maintainers can appear to if there is disagreement that the maintainers cannot deal with on their own, and to address some cross cutting concerns (e.g. security issue reporting)

Within the docs, the mission of AC sounds a little bit more active and interesting:

> The openHAB Architecture Council (AC) serves the community by identifying and tackling any issues that hinder openHAB’s continued technological success and innovation, widespread adoption, and future growth. This involves technical architecture as well as open source processes and social aspects. Comprising the finest technical leaders from all community stake holders, it is the council’s goal to keep the project successful and healthy, the processes simple and smooth, and the communities vibrant and cohesive.

If somebody from outside who hasn't access to this page reads this, he has a completely different picture than you described. Maybe we should adjust this description, wdyt?

As I wrote already within the docs issue: Staying as it is, the explanation of the AC just leaves a big questionmark for people who don't have access. They could misinterpret it as a very active, "secret" gremium which decides anything of importance behind the scenes. If you have access, you see that discussions & decisions by the AC usually are barely relevant for users, they really focus on very technical software development aspects of the project. But if it's a blackbox, a user may get a completely different impression and thinks he cannot participate in many discussions that are very relevant for him.
